tempest/api/workloadmgr/snapshot/test_snapshot_mount.py

```diff
@@ -530,7 +540,7 @@ def test_7_snapshot_mount_unmount_full_imagebooted_fvm_cli(self):
             else:
                 reporting.add_test_step(
                     "Snapshot unmount of full snapshot", tvaultconf.FAIL)
-            tests[1][1] = 1
+            tests[2][1] = 1
```

Comment:
Whats the purpose of setting these values to 1 if we are not going to use it at the end for recording whether particular test case is passed or not like below,
for test in tests:
                if test[1] != 1:
                    reporting.set_test_script_status(tvaultconf.FAIL)
                    reporting.add_test_script(test[0])
                    reporting.test_case_to_write()